flybywiresim/a32nx
# fix(ap): ALT CST* and ALT CST conditions and transitions

## Summary of Changes
This PR fixes an issue when in ALT CST* and the altitude constraint is changing but matches FCU target altitude. In that case ALT* engaged even if the ALT* condition is not satisfied. Now CLB or DES will correctly engage in the same situation.

Additionally, a time delay has been added to ALT CST condition as it was added to ALT condition before.

## Testing instructions
This is hard to test manually because one needs to be exactly in the ALT CST* condition for the issue to be triggered. I would test if any other unwanted effects are seen. I tested it manually with the model running outside the sim.

Beside that, the way to test it is:
- engage NAV
- fly at 5000 ft
- set altitude constraint to 6000 ft
- set FCU altitude to 7000 ft
- engage CLB
- when in ALT CST* set altitude constraint to 7000 ft
- expected result: CLB engages with ALT armed


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
